WORKSPACE

```diff
@@ -67,12 +64,31 @@ bind(
     actual = "//tools:java_toolchain",
 )
 
+http_archive(
+    name = "rules_java",
+    url = "https://github.com/bazelbuild/rules_java/releases/download/0.1.1/rules_java-0.1.1.tar.gz",
+    sha256 = "220b87d8cfabd22d1c6d8e3cdb4249abd4c93dcc152e0667db061fb1b957ee68",
+)
+
 load("@rules_java//java:repositories.bzl", "rules_java_dependencies", "rules_java_toolchains")
 
 rules_java_dependencies()
 
 rules_java_toolchains()
```

Comment:
Suggest removing newlines for consistency.

Reply:
Done